SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityNewTaskChecker.java

```diff
@@ -300,6 +300,7 @@ private CheckTaskState getTaskState(SingularityTask task) {
   private Optional<CheckTaskState> checkLbState(BaragonRequestState lbState) {
     switch (lbState) {
       case SUCCESS:
+      case INVALID_REQUEST_NOOP:
         return Optional.of(CheckTaskState.HEALTHY);
```

Comment:
in my original notes, INVAILD_REQUEST_NOOP should be fired when removing a nonexistent upstream, or removing a service from a nonexistent LB -- let's sync up if that state is supposed to mean something else